Resources/Audio/_Goobstation/Items/Fishing/attributions.yml

```
@@ -0,0 +1,9 @@
+- files: ["fishing_rod_cast.ogg"]
+  license: "CC-BY-NC-SA-4.0"
+  copyright: "Taken from Goon at commit 0c11a46275f493f295d6d05d1aa5fad1d935efb4"
+  source: "https://github.com/goonstation/goonstation/blob/master/sound/items/fishing_rod_cast.ogg"
+  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)